src/main/java/core/basesyntax/dto/FruitTransactionDto.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.dto;
+
+import java.util.Objects;
+
+public class FruitTransactionDto {
+    private String operationType;

```

Comment:
BTW, this field should have been an enum type here, this is not in scope of this task anymore, but just so that you know for the future

Reply:
Ok, thank you!